### DISCUSSION TITLE
Textlocal sms otp not getting sent from Supabase |  supabase.auth.signInWithOtp api | OTP authentication

### DISCUSSION THREAD

**Discussion by vasanth-ravisankar** [supabase/supabase#15350] [Questions]
I'm trying to integrate textlocal sms service for phone number otp authentication. But the otp is not getting sent to the user when we trigger the supabase.auth.signInWithOtp api. 

But when I hit the textlocal api to send SMS with the same textlocal account and configurations, the SMS is getting sent as expected.

Supabase documentation on textlocal integrations is also very limited.

Have anyone faced this or similar issue already. Please help me to solve this issue. Giving the error message I found in Supabase auth logs.

{"auth_event":{"action":"user_recovery_requested","actor_id":"3725fc5e-6839-4e44-a982-06dd0e4ee6ee","actor_username":"919487073472","actor_via_sso":false,"log_type":"user","traits":{"channel":"sms"}},"component":"api","error":"400: Error sending sms OTP: \u003cnil\u003e","level":"info","method":"POST","msg":"400: Error sending sms OTP: \u003cnil\u003e","path":"/otp","referer":"","remote_addr":"103.87.106.7","time":"2023-06-26T07:41:41Z","timestamp":"2023-06-26T07:41:41Z"}

**Comment by J0** [comment]
Hey @vasanth-ravisankar,

Thanks for reporting the issue,  and sorry to hear that you are facing issues with the provider.  You are right that there should probably be docs on how to use the provider. Do you mind sharing the exact API request made? Am wondering if the endpoint has changed as mentioned earlier in this thread https://github.com/supabase/gotrue/pull/342#issuecomment-1514651066

**Reply by vasanth-ravisankar** [reply]
Hi @J0 ,
Thanks for responding to my request. Providing you the API request I'm making to get an login SMS OTP.

```
await supabase.auth.signInWithOtp({
   phone: `+91${mobileNumber}`,
  });
  ```
